test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -218,13 +224,17 @@ std::string RandomCode::generate(int _maxOpNumber, RandomCodeOptions _options)
 	for (auto i = 0; i < size; i++)
 	{
 		uint8_t opcode = weightsDefined ? randOpCodeWeight() : randOpCodeGen();
-		dev::eth::InstructionInfo info = dev::eth::instructionInfo((dev::eth::Instruction) opcode);
+		eth::Instruction inst = (eth::Instruction) opcode;
+		eth::InstructionInfo info = eth::instructionInfo(inst);
 
-		if (info.name.find("INVALID_INSTRUCTION") != std::string::npos)
+		if (info.name.find("INVALID_INSTRUCTION") != std::string::npos || info.name.empty()
+			|| std::find(invalidOpcodes.begin(), invalidOpcodes.end(), inst) != invalidOpcodes.end())
 		{
-			//Byte code is yet not implemented
-			if (_options.useUndefinedOpCodes == false)
+			if (_options.useUndefinedOpCodes)
+				code += toCompactHex(opcode, HexPrefix::DontAdd, 1);
```

Comment:
there one sense: to see what would happen. so if flag is defined it will randomly add a byte that does not represent a valid opcode. that was the initial idea.